base/system/services/database.c

```diff
@@ -852,6 +852,105 @@ ScmDeleteServiceRecord(PSERVICE lpService)
     DPRINT("Done\n");
 }
 
+DWORD
+Int_EnumDependentServicesW(HKEY hServicesKey,
+                           PSERVICE lpService,
+                           DWORD dwServiceState,
+                           PSERVICE *lpServices,
+                           LPDWORD pcbBytesNeeded,
+                           LPDWORD lpServicesReturned);
```

Comment:
Glad to see someone finally approaching that long-standing issue. Still a few formal suggestions:
```suggestion
Int_EnumDependentServicesW(
    HKEY hServicesKey,
    PSERVICE lpService,
    DWORD dwServiceState,
    PSERVICE *lpServices,
    LPDWORD pcbBytesNeeded,
    LPDWORD lpServicesReturned);
```

Reply:
He kept the formatting of the original function implemented in the other file.